sftp-source.kamelet.yaml

```diff
@@ -0,0 +1,68 @@
+    camel.apache.org/provider: "Apache Software Foundation"
+  labels:
+    camel.apache.org/kamelet.type: "source"
```

Comment:
Should we add label camel.apache.org/kamelet.group=FTP in this and the FTP case?
 In case they belong to the same group we should also use the same icon, so that we can group them together in the visualization. Wdyt?

Reply:
Sounds good, do we want to use the ftp icon for both?

Reply:
Yes, it will look odd at first, but we can change the website style later to add support for groups and fix it.